ArtOfIllusion/src/artofillusion/math/Vec3.java

```diff
@@ -178,6 +178,20 @@ public final void normalize()
         z /= len;
       }
   }
+  
+  /** Crate a copy of this vector, scaled to the length of 1. If the length of this vector is 0, 
```

Comment:
I thought it might be handy.... :)

This thing is genetic, and the usual "be more careful" and "read it once more over" don't do it for me. It is not uncommont, that when I write a few lines of code, it takes more time of Ant to get through the spell checking phase that I took me to write it. -- So, what can I but to add one more of the usual comments: "Be patient...." :)